enhancements/olm/operator-bundle.md

```diff
@@ -105,17 +105,30 @@ An operator author can also specify the version of the format used inside the bu
 operators.operatorframework.io.bundle.mediatype: "registry+v1"
 ```
 
+###### Graph Metadata
+An additional point of concern is the fact that the current [operator-registry manifest format](https://github.com/operator-framework/operator-registry#manifest-format) also has a multi-version aggregation file `package.yaml` that describes data about the channels that individual versions are subscribed to. Given that we are splitting up these manifests into separate objects, we now need a way to define how an individual version subscribes to a given channel. To do this, we will create a new metadata file that defines the channels explicitly for each version called `channels.yaml`
+
+*channels.yaml*
+```yaml
+channels:
+ - alpha
+ - stable
+```
+
+Also note that this file is optional. If it is not specified, the registry tooling will put the bundle in a default channel.
```

Comment:
by default channel do you mean a new channel named default or is there some other mechanism to specify the default channel?

Reply:
sorry, maybe this isn't clear. I'm not suggesting we are defining the `default` channel. i'm suggesting that if you don't specify a channel then when that operator bundle is added to an index we will have a predefined default channel for you. I was thinking that channel can be called `default`.

Reply:
So what if an older version of the bundle had a default channel defined? 

would you:
1. Create a new channel `default`
2. Use the existing channel that was marked as default most recently
3. Fail(?) 
4. ? some other option that I can not think of

What happens if I add a default AFTER I have been using this default? do we move the implicit default channel to this newly named default channel? Do we keep the default channel and the new default channel and mark the new one `default` even if there is a named "defautl" channel? 

I think we need to describe the rules around this.

Reply:
Yep we do need to define all of it. In this enhancement I was just calling out that this file was optional. In the operator-registry enhancement I am going to create an additional PR that will describe how all of this will work.

Reply:
@shawn-hurley I am going to back off on this for now and will define what I am imagining for this defaulting behavior in a future enhancement.